### PR TITLE
Add battle-bot game

### PR DESCRIPTION
**Pull request by thekyleliao**
2 player game where players attempt to push each other into death zones whilst avoiding them.

### CODE REVIEW

**Review by LucasHT22 — COMMENTED** [review]
Thanks for PR!

File: games/battle bots.js [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+3333333333333333
+3333333333333333
+3333333333333333
+3333333333333333
+3333333333333333
+3333333333333333
+3333333333333333
+3333333333333333
+3333333333333333` ],
+    [ wall, bitmap`
```
Comment (LucasHT22) [inline games/battle bots.js:102]: When is Wall used?

**Comment by thekyleliao** [comment]
I was using it beforehand. It's not currently used in the current map
config, but I might add in walls if playing doesn't push players close
enough together.

**Comment by LucasHT22** [comment]
do you prefer to add to current configuration or delete?

**Comment by thekyleliao** [comment]
Delete?

**Comment by thekyleliao** [comment]
Updated game to remove other wall sprite. Also, updated for new color system.

**Review by LucasHT22 — APPROVED** [review]
Congrats!
If you're a teen, you can get your sprig here! https://sprig-order.hackclub.dev/

**Pull request merged**